### PR TITLE
Light Signup v3.0.0

### PR DESCRIPTION
**Pull request by tavvy**
no whitespace: https://github.com/Financial-Times/next-article/pull/1419/files?w=1

screen shot 2016-06-01 at 16 45 40

### CODE REVIEW

File: views/partials/light-signup.html [hunk 1]
```diff
@@ -1,24 +1,53 @@
...
-					<input data-o-grid-colspan="7" type="text" id="email" name="email" class="o-email-only-signup__email o-forms-text" data-trackable="email">
-					<input type="hidden" name="source" value="next">
-					<input type="hidden" name="mailingList" value="next-article">
-					<input type="hidden" name="topic" value="default">
-					<button data-o-grid-colspan="4" type="submit" class="o-email-only-signup__submit" data-trackable="submit">Sign up</button>
-					<p class="o-forms-errortext o-email-only-signup__email-error-msg o-email-only-signup__visually-hidden" data-o-email-only-signup-email-error>Invalid email</p>
-					<p class="o-email-only-signup__no-spam">By clicking Sign up you confirm that you have read and agree to the <a target="_blank" href="http://help.ft.com/tools-services/ft-com-terms-and-conditions">terms and conditions</a>, <a target="_blank" href="http://help.ft.com/tools-services/how-the-ft-manages-cookies-on-its-websites">cookie policy</a> and <a target="_blank" href="http://help.ft.com/tools-services/financial-times-privacy-policy">privacy policy</a>.<br /><strong>Unsubscribe at any time.</strong></p>
-				</form>
+{{#if @root.flags.lightSignupInArticle}}
```
Comment (andygnewman) [inline views/partials/light-signup.html]: I would suggest putting the if condition in content.html https://github.com/Financial-Times/next-article/blob/master/views/content.html#L42 You can use an #ifAll handlebars helper as is used https://github.com/Financial-Times/next-article/blob/master/views/content.html#L79

Comment (tavvy) [inline views/partials/light-signup.html]: This is great, I had been searching for some kind of if all functionality!

**Comment by tavvy** [comment]
Feedback: https://github.com/Financial-Times/next-article/pull/1419/commits/ac18d32471c7693b6fdfb4e5b0aef557c7718874?w=1

File: views/partials/light-signup.html (hunk 1 above)
Comment (bjfletcher) [inline views/partials/light-signup.html]: Useful bookmark for helpers: https://github.com/Financial-Times/n-handlebars/blob/master/src/extend-helpers.js

**Comment by andygnewman** [comment]
If you're fixing the o-buttons version resolution can you use 3.0.0 not 4.0.0

**Comment by ironsidevsquincy** [comment]
👍

**Pull request merged**